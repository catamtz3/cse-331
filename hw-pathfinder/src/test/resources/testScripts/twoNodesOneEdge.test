CreateGraph twoNodes

AddNode twoNodes a
AddNode twoNodes b

AddEdge twoNodes a b 10.0
AddEdge twoNodes a c 5.0

ListNodes twoNodes

ListChildren twoNodes a

FindPath twoNodes a b