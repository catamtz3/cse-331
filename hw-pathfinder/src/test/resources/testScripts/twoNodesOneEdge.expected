created graph twoNodes

added node a to twoNodes
added node b to twoNodes

added edge 10.000 from a to b in twoNodes
added edge 5.000 from a to c in twoNodes

twoNodes contains: [a, b]

the children of a in twoNodes are: b(10.000)

path from a to b:
a to b with weight 10.000
total cost: 10.000